operations/CSM_product_management/Perform_NCN_Personalization.md

```diff
@@ -41,8 +41,8 @@ error. This error can be ignored.
 ### Add Layer(s) to the CFS Configuration
 
 CFS executes configuration layers in order. Refer to the _1.5 HPE Cray EX
```

Comment:
Should "1.5" be included here? For Papaya, the guide is version 22.11 or edition 5. Is this a reference to an older version of the guide that should be removed?

```suggestion
CFS executes configuration layers in order. Refer to the _HPE Cray EX
```